### ISSUE TITLE
[Bug] Accessibility: Undo Close disappearing too fast.

### ISSUE THREAD

**Issue by MarcoZehe** [mozilla-mobile/fenix#8998]
## Steps to reproduce

1. Open Fenix with TalkBack.
2. Open a tab with any arbitrary URL.
3. Close the tab.

### Expected behavior

User should have a chance to locate the Undo Close button.

### Actual behavior

Button disappears roughly when TalkBack finishes speaking, and is not to be found again.

### Device information

* Android device: ? Pixel 3.
* Fenix version: ? Nightly.

## More info

The best would be a sub menu in the main menu popup that holds the recently closed tabs, much like the Recently Closed Tabs sub menu in the History menu of Firefox Desktop.

CC @eeejay

**Comment by sv-sdeiac** [comment]
Hi @MarcoZehe., I was not able to reproduce this issue using Google Pixel (Android Q) on the latest Firefox Preview Nightly 3/5 #20650606.

The time for the closed tab is about ~15 sec.
Can you please retry with the latest version of Firefox Preview Nightly? If you use some specific options, please let me know, thanks.

**Comment by MarcoZehe** [comment]
I can still reproduce this, it disappears quite fast. Only if I immediately double-tap after closing the tab, I might get it back, but this is far from reliable.

**Comment by sblatz** [comment]
Hmm this is really strange. I specifically implemented a fix for this to stay for 15 seconds (in collaboration with @eeejay) when TalkBack is enabled. @MarcoZehe what nightly version are you on?

**Comment by asadotzler** [comment]
I just tested this on Fenix Nightly 3/28 and the undo banner and button lasted about 15 seconds which was just enough time for me to quickly swipe through all of the homepage content to get to the button and select Undo.

**Comment by asadotzler** [comment]
It would be nice if the Undo banner had focus if it didn't disappear. On a second attempt I was a bit slower but made it to the button just in time for the button to be read out to me and then for the banner to disappear before I could activate the button.

**Comment by asadotzler** [comment]
This is an accessibility P2. The feature is available and works but is difficult to use because of the time out and the lack of persistence on focus. This kind of bug prevents Fenix from being a delightful experience for users with disabilities.

**Comment by eeejay** [comment]
I think the 15 seconds is an improvement over the default timeout, but not perfect. I think it would be useful if the accessibility focus went to the undo button, and when it goes away go to the next open tab. I think it would not be entirely straightforward to implement, but not impossible.

Chrome has a similar issue with their reopen closed tab button. It only appears for a few seconds.

**Comment by eliserichards** [comment]
Closing in favor of #10047
